### DISCUSSION TITLE
Given parent table not found in system catalogs: public.action_items_partitioned

### DISCUSSION THREAD

**Discussion by jdesouza** [pgpartman/pg_partman#588] [General]
Running select partman.run_maintenance();
We get:

ERROR:  Given parent table not found in system catalogs: public.table1_partitioned
CONTEXT: PL/pgSQL function show_partitions(text,text,boolean) line 51 at RAISE
SQL statement "SELECT partition_tablename                                   FROM partman.show_partitions(v_row.parent_table, p_include_default := true) LIMIT 1"
PL/pgSQL function partman.run_maintenance(text,boolean,boolean) line 162 at SQL statement
DETAIL: 
HINT: 
CONTEXT:  PL/pgSQL function partman.run_maintenance(text,boolean,boolean) line 413 at RAISE
SQL state: P0001

public.table1_partitioned was renamed to public.table1 because this table was replacement for non partitioned public.table1.

any idea what could be going on?

**Comment by keithf4** [comment]
What is the result of the following query when connected to the database that you're running maintenance from? 
```
SELECT c.relname, n.nspname FROM pg_class c JOIN pg_namespace n ON c.relnamespace = n.oid WHERE relname = 'table1_partitioned';
```

**Comment by jdesouza** [comment]
SELECT c.relname, n.nspname FROM pg_class c JOIN pg_namespace n ON c.relnamespace = n.oid WHERE relname = 'table1_partioned'; returns no data

Only returns data by the renamed name:
SELECT c.relname, n.nspname FROM pg_class c JOIN pg_namespace n ON c.relnamespace = n.oid WHERE relname = 'table1';

**Comment by keithf4** [comment]
Look for an entry in the `part_config` table for `table1_partioned`.

If you've renamed your partition table's parent table name, you'll have to update that entry in the `part_config` table. If you have a template table as well and renamed that, you'll have to update that entry in the `part_config` table as well.

**Reply by jdesouza** [reply]
Thanks!
